dbt/include/databricks/macros/materializations/snapshot.sql

```diff
@@ -58,7 +73,7 @@
 
       {{ adapter.valid_snapshot_target(target_relation) }}
 
-      {% set staging_table = spark_build_snapshot_staging_table(strategy, sql, target_relation) %}
+      {% set staging_table = databricks_build_snapshot_staging_table(strategy, sql, target_relation) %}
```

Comment:
Not urgent but should we use the dispatch pattern here to make it backward compatible?